timeline_logger/management/commands/prune_timeline_logs.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+from textwrap import dedent
+
+from django.core.management.base import BaseCommand
+
+from timeline_logger.models import TimelineLog
+
+
+class Command(BaseCommand):
+    help = "Removes timeline logs objects older than the specified date."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--noinput",
+            "--no-input",
+            action="store_false",
+            dest="interactive",
+            help="Tells Django to NOT prompt the user for input of any kind.",
+        )
+        parser.add_argument(
+            "--before",
+            type=datetime.fromisoformat,
+            help="Only flush timeline logs older than the specified date.",
+        )
+
+    def handle(self, *args, **options):
```

Comment:
Please implement the business logic in a standalone function instead of directly in the management command, so that we can add other entrypoints to this functionality like:

* a celery task (scheduled via beat)
* possibly a view

I also think that providing a timestamp is not the preferred approach, but instead something like `--max-age 90d` makes more sense - keep everything from the last 90 days, discard everything older than that.